src/libraries/System.Private.Uri/tests/FunctionalTests/UriRelativeResolutionTest.cs

```diff
@@ -617,11 +617,11 @@ public void UriRelative_AbsoluteToAbsolute_CustomPortCarriedOver()
             Uri baseUri = new Uri("http://nothing.com/");
             Uri testUri = new Uri("https://specialPort.com:00065535/path?query#fragment");
 
-            int throwAway = testUri.Port; // Trigger parsing.
+            _ = testUri.Port; // Trigger parsing.
 
             Uri resultUri = new Uri(baseUri, testUri);
 
-            throwAway = resultUri.Port; // For Debugging.
+            _ = resultUri.Port; // For Debugging.
```

Comment:
This looks exactly duplicated from the product code.  What's going on here?

Reply:
I removed these as the evaluation is forced in the assert below regardless. Were you referring to something else?

Reply:
I was referring to this code in tests being identical to code I saw earlier in the PR in product source.  If you deleted this, great.